source/frontend/src/features/properties/map/agreement/detail/AgreementContainer.tsx

```diff
@@ -5,6 +5,7 @@ import { useEffect } from 'react';
 import { useCallback } from 'react';
 import { useContext } from 'react';
 
+import { useGenerateAgreement } from '../../acquisition/common/GenerateForm/hooks/useGenerateAgreement';
```

Comment:
nit: this looks like a smell that we should move the hooks to somewhere more common